ironfish/src/workerPool/tasks/mineHeader.ts

```diff
@@ -8,7 +8,7 @@ import { mineHeader } from '../../mining/mineHeader'
 export type MineHeaderRequest = {
   type: 'mineHeader'
   batchSize: number
-  headerBytesWithoutRandomness: Uint8Array
+  headerBytesWithoutRandomness: Buffer
```

Comment:
NodeJS uses [object cloning](https://nodejs.org/api/worker_threads.html#considerations-when-cloning-objects-with-prototypes-classes-and-accessors) when transferring data to a worker thread, so `headerBytesWithoutRandomness` will end up as a `Uint8Array` when the MineHeaderRequest is serialized and sent to the worker.

So for this reason, I'd like to leave this type as `Uint8Array`. However, in `mining/mineHeader`, I think that type can be changed to `Uint8Array`, and then we can avoid the same allocation.

Reply:
Thanks for the explanation. Done with the changes.